src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,11 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService userService = new UserService();
+        if (userService.findByEmail(email) == null) {
+            return false;
+        }
```

Comment:
call findByEmail only once in this method, it may be a costly operation, save the result of the call to a local variable
try to rewrite your method without using true/false literals